editorial_material/why_livecoms.tex

```diff
@@ -73,16 +73,17 @@ \section{Preprints, perpetual/living reviews, and overlay journals provide conte
 Thus, preprints provide a glimpse of a path forward, as they are updatable documents which are current and have real value.
 Posting of preprints is also typically free, lowering the entry barrier. 
-However, in much of CoMS (Physics being a potential exception) they do not yet provide the same degree of academic credit as a peer-reviewed publication. 
+However, in much of computational molecular science (some areas of physics being a potential exception) preprints do not yet provide the same degree of academic credit as a peer-reviewed publication. 
 
 Another glimpse is provided by the concept of ``perpetual'' or ``living'' reviews [ref Zuckerman/Mobley and other references therein]; these begin as traditional review papers but the authors keep them current by releasing new versions regularly which incorporate new developments and/or community feedback.
 These provide the potential that an author who stays in a given field could maintain an authoritative, current document which always reflect the state of the field and the author(s)' views. 
-One of the present authors (DLM) is currently experimenting with just such a review [ref], and it has already expanded significantly to incorporate new developments, even gaining an additional author.
+Two of the present authors (DLM) are currently experimenting with just such a review [ref], and it has already expanded significantly to incorporate new developments, even gaining an additional author.
 
 Still, neither of these advancements provide a mechanism for publication credit; this type of credit can be provided by the concept of ``overlay journals''.
 Overlay journals are journals which don't formally host articles themselves, but which provide peer reviews of specific versions of preprints, then, based on these reviews, publishes articles by link to them and giving them status as formal publications.
-Examples of such publications include Discrete analysis [ref website: http://discreteanalysisjournal.com/ and blog explanation: https://gowers.wordpress.com/2015/09/10/discrete-analysis-an-arxiv-overlay-journal/], Quantum [http://quantum-journal.org/instructions/authors/], and  [add others].
+Examples of such publications include Discrete Analysis [ref website: http://discreteanalysisjournal.com/ and blog explanation: https://gowers.wordpress.com/2015/09/10/discrete-analysis-an-arxiv-overlay-journal/], Quantum [http://quantum-journal.org/instructions/authors/], and  [add others].
 This allows fairly formal publication credit; for example, both are indexed by Google Scholar [confirm], and Quantum has an ISSN and is working towards inclusion in Web of Science and other indices. 
+%MRS: ISSN is trival to get. Not even worth discussing here.  In theory the articles are indexed in Google Scholar, but I'm having a hard time finding randomly selected articles - I get their arxiv version instead, with no journal website.  I have a query into scholastica.
```

Comment:
OK. They reported it as if it was a milestone, so I assumed it was. But I'll remove.